bazel/repository_locations.bzl

```diff
@@ -767,11 +767,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "wasmtime",
         project_desc = "A standalone runtime for WebAssembly",
         project_url = "https://github.com/bytecodealliance/wasmtime",
-        version = "0.39.1",
-        sha256 = "6ef70886da14245f575c6ff8c7c999ae22579257eba5ebf382e066598c1e381c",
+        version = "1.0.2",
```

Comment:
Again, this only updates the Wasmtime C API (i.e. the glue code), the core of Wasmtime is what Proxy-Wasm C++ Host points at, which is still v0.39.1 in case of commit in this PR.

Reply:
arggh - i thought i had updated proxy-wasm-c++-host ill check this again ...

Reply:
There is no v1.0.2 in Proxy-Wasm C++ Host, we have v1.0.1 and v2.0.0 at the HEAD.

Reply:
k - i see - can we update there - i think the change from 1.0.1 is just the CVE fix/es

Reply:
Considering that Wasmtime was already updated in the stable release, it would be easier to go straight to v2.0.2, since we don't maintain old branches in Proxy-Wasm C++ Host, and HEAD is already at v2.0.0. Could you send a PR?

Reply:
i can create a PR but `master` is beyond this now - im wondering can we create a `wasmtime-1.0.2` branch ?

Reply:
k - yep - ill update -> 2.0.2